backend/src/test/java/com/woowacourse/momo/member/service/MemberServiceTest.java

```diff
@@ -78,30 +79,54 @@ void findById() {
     @Test
     void updatePassword() {
         Long memberId = createMember();
-        Member beforeMember = memberRepository.findById(memberId).get();
+        Member beforeMember = memberFindService.findMember(memberId);
         String beforePassword = beforeMember.getPassword();
 
-        ChangePasswordRequest request = new ChangePasswordRequest("wooteco2!");
+        PasswordRequest request = new PasswordRequest("wooteco2!");
         memberService.updatePassword(memberId, request);
 
-        Member member = memberRepository.findById(memberId).get();
+        Member member = memberFindService.findMember(memberId);
         assertThat(member.getPassword()).isNotEqualTo(beforePassword);
     }
 
     @DisplayName("회원 이름을 수정한다")
     @Test
     void updateName() {
         Long memberId = createMember();
-        Member beforeMember = memberRepository.findById(memberId).get();
+        Member beforeMember = memberFindService.findMember(memberId);
         String beforeName = beforeMember.getName();
 
         ChangeNameRequest request = new ChangeNameRequest("무무");
         memberService.updateName(memberId, request);
 
-        Member member = memberRepository.findById(memberId).get();
+        Member member = memberFindService.findMember(memberId);
         assertThat(member.getName()).isNotEqualTo(beforeName);
     }
 
+    @DisplayName("올바른 비밀번호인지 확인한다")
+    @Test
+    void confirmPassword() {
+        String password = "wooteco1!";
+        SignUpRequest signUpRequest = new SignUpRequest("woowa", password, "모모");
+        Long memberId = authService.signUp(signUpRequest);
+
+        PasswordRequest passwordRequest = new PasswordRequest(password);
+        assertDoesNotThrow(() -> memberService.confirmPassword(memberId, passwordRequest));
+    }
+
+    @DisplayName("잘못된 비밀번호면 예외가 발생한다")
```

Comment:
이 설명만으로는 테스트 돌릴 때 어떤 테스트에서 실패하는지 바로 파악하기 어려울 것 같아요. 어떤 행위를 하는지 앞에 추가하는 것은 어떨까요?

```suggestion
    @DisplayName("비밀번호 확인 시 입력한 값이 잘못된 비밀번호면 예외가 발생한다")
```

Reply:
좋습니다! 바로 반영해보겠습니다